release-notes/2023-11/README.md

```diff
@@ -1,31 +1,37 @@
+    - The migration requires permission for `ALTER TABLE`, which is an elevated privileges beyond the minimal set required by the service's normal operation. To use `ALTER TABLE` the PostgreSQL user must be the owner of the table or be a member of the role that owns the table. If your PostgreSQL user does not have the required permissions the updated pods job will fail to start. See below for new options to facilitate this requirement.
 
-- nbexecservice
-    - Notebooks are now running on Python 3.11.
+### Setting up an Elevated PostgreSQL User for Migration
 
-- testmonitorservice 0.16.37
-    - The update includes a new database schema migration to support future features. This migration adds a new column to the `results` table. It is expected to complete without downtime. Downgrades to prior versions after updating to this version is not supported.
-    - The migration requires permission to ALTER TABLE, which is an elevated privileges beyond the minimal set needed by the service. To ALTER TABLE the PostgreSQL user must be the owner of the table, or be a member of the role that owns the table. If your PostgreSQL user does not have the required permissions the updated pods will fail to start. See below for new options to facilitate this requirement.
-    - Database migrations are now managed by a Kubernetes Job that runs prior to deploy of the TestMonitor pods. This will allow rolling updates to TestMonitor to be peformed more consistently.
-    - The testmonitorservice helm chart includes new options to specify the PostgreSQL user used for migration separately from the user used by the service. This allows a lower privileged user to be used day-to-day by the service while a higher privileged user is only used during application updates.
-    - The new helm chart options are `testmonitorservice.secrets.database.migrationConnectionString` and `testmonitorservice.secrets.database.migrationConnectionPassword` in the secrets values file [View this configuration](https://github.com/ni/install-systemlink-enterprise/blob/2023-11/getting-started/templates/systemlink-secrets.yaml#L434), and `testmonitorservice.database.connectionString.migrationConnectionStringKey`, `testmonitorservice.database.connectionInfo.migrationUser`, and `testmonitorservice.database.connectionInfo.migrationPasswordKey` in the values file [View this configuration](https://github.com/ni/install-systemlink-enterprise/blob/2023-11/getting-started/templates/systemlink-values.yaml#L287).
+- The database migrations is performed by a Kubernetes job that runs prior to TestMonitor pod deployment.
+- The `testmonitorservice` Helm chart includes a new options to specify the PostgreSQL user for migration separately from the user for the service's normal operation. This allows the higher privileged user to be used during application updates exclusively.
```

Comment:
We might want to call out that setting a separate user for migration is optional.

Reply:
I agree, these read like these new configuration options are required, which they are not. If you only use the prior configuration that will apply to both to the service replicas and the migration.

I also have a PR to update the ni.com docs with detailed instructions on how to configure these user credentials. That needs review from @MelissaHilliard . If that update is approved we should link to that document here. It will be an update to [this document](https://www.ni.com/docs/bundle/systemlink-enterprise/page/config-systemlink-enterprise.html#GUID-22D7F822-FF82-436A-9458-DA1D33334886__GUID-A2CA66A5-7E59-419B-8638-48E7E7A3963A).

[Pull Request 590739](https://dev.azure.com/ni/Users/_git/Documentation%20Reviews/pullrequest/590739): Expand docs on configuring postgres users for TestMonitor

Reply:
@MelissaHilliard do you think this document would be published this week so we could refer to it in the release notes? Do you think we should lift the current wording and structure that will be used in the ni.com help into the release notes?